packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCard/AddCard.types.ts

```diff
@@ -0,0 +1,61 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IAddCardProps {
+  id: string;
+  /**
+   * The title for card representation in add card panel
+   */
+  title: string;
+
+  /**
+   * The image url for card representation in add card panel
+   */
+  imageSrc?: string;
+
+  /**
+   * The description for card representation in add card panel
+   */
+  description?: string;
+
+  /**
+   * The description for card representation in add card panel
+   */
+  cardClicked?: (cardId: string) => void;
```

Comment:
>cardClicked [](start = 2, length = 11)

this is addCard right not card click, more informative would be clickToAddCard

Reply:
This again is not exposed to user, this is one individual card representation. The callback just gives it's ID back when it's clicked, which is why I thought the cardClicked would be appropriate as it defines the action and this is the callback received upon that action.